### DISCUSSION TITLE
Github student verification failed due to 2FA were not set up

### DISCUSSION THREAD

**Discussion by Yukiwang77** [community/community#115006] [GitHub Education]
### Hi everyone,

Hi there,
I was in process of verifying my student status. I uploaded my student ID but it errored out after the photo got processed (passed) due to 2FA were not set up at the time. I set 2FA up, and came back to benefit and tried to resubmit my student photo id as supporting document but it says the document has been uploaded. What should I do from here? dont think the student benefit got granted
Thank you in advance!

**Comment by samus-aran** [comment]
Hi @Yukiwang77 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [Your uploaded proof has already been used previously.](https://github.com/orgs/community/discussions/111352#uploaded-proof-already-used)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
